### PR TITLE
[chrome] Add chrome.enterprise.networkingAttributes and update chrome.platformKeys

### PR DESCRIPTION
**Pull request by mpetrisor**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
  - I believe no tests are necessary for this change. 
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
https://developer.chrome.com/extensions/enterprise_networkingAttributes
https://developer.chrome.com/extensions/platformKeys#method-getKeyPairBySpki

### CODE REVIEW

File: types/chrome/index.d.ts [hunk 1]
```diff
@@ -5079,6 +5104,17 @@ declare namespace chrome.platformKeys {
...
+     * @param parameters Provides signature and hash algorithm parameters, in addition to those fixed by the key itself. The same parameters are accepted as by WebCrypto's importKey function, e.g. RsaHashedImportParams for a RSASSA-PKCS1-v1_5 key. For RSASSA-PKCS1-v1_5 keys, we need to also pass a "hash" parameter { "hash": { "name": string } }. The "hash" parameter represents the name of the hashing algorithm to be used in the digest operation before a sign. It is possible to pass "none" as the hash name, in which case the sign function will apply PKCS#1 v1.5 padding and but not hash the given data.
+     * Currently, this function only supports the "RSASSA-PKCS1-v1_5" algorithm with one of the hashing algorithms "none", "SHA-1", "SHA-256", "SHA-384", and "SHA-512".
+     * @param callback The public and private CryptoKey of a certificate which can only be used with platformKeys.subtleCrypto.
+     * The callback parameter should be a function that looks like this:
+     * function(object publicKey, object privateKey) {...};
+     * Optional parameter privateKey: Might be null if this extension does not have access to it.
+     */
```
Comment (pokutuna) [inline types/chrome/index.d.ts:5116]:
```suggestion
     * @since Chrome 85.
     */
```

File: types/chrome/index.d.ts [hunk 2]
```diff
@@ -2588,6 +2588,31 @@ declare namespace chrome.enterprise.deviceAttributes {
     export function getDeviceAnnotatedLocation(callback: (annotatedLocation: string) => void): void;
 }
 
+////////////////////
+// Enterprise Networking Attributes
+////////////////////
+/**
+ * Use the <code>chrome.enterprise.networkingAttributes</code> API to read information about your current network. Note: This API is only available to extensions force-installed by enterprise policy.
+ */
```
Comment (pokutuna) [inline types/chrome/index.d.ts:2596]:
```suggestion
 * Important: This API works only on Chrome OS.
 * @since Chrome 85.
 */
```

**Review by pokutuna — CHANGES_REQUESTED** [review]
Looks good.
I suggested doc blocks improvements.

**Comment by mpetrisor** [comment]
Hello @pokutuna,

Thank you for your suggestions! I have committed them.

mpetrisor

**Review by pokutuna — APPROVED** [review]
Thanks for merging my suggestions. LGTM

**Pull request merged**